Cabal/Distribution/InstalledPackageInfo.hs

```diff
@@ -73,17 +75,19 @@ import Data.Set (Set)
 data InstalledPackageInfo
    = InstalledPackageInfo {
         -- these parts are exactly the same as PackageDescription
-        -- | Traditionally, 'sourcePackageId' records the 'PackageId'
-        -- of the package associated with this library, and most tooling
-        -- assumes that this field uniquely identifies any package
-        -- that a user might interact with in a single GHC session.
-        -- However, with convenience libraries, it's possible for there
-        -- to be multiple libraries associated with a package ID; to
-        -- keep backwards compatibility with old versions of GHC,
-        -- 'sourcePackageId' actually stores a *munged* version of the
-        -- package identifier that also incorporates the component name.
-        -- The /real/ package name is stored in 'sourcePackageName'.
-        sourcePackageId   :: PackageId,
```

Comment:
Well, it depends on whether or not sourcePackageId returns a PackageId or a MungedId ;)